src/cloudflare/internal/vectorize-api.ts

```diff
@@ -68,7 +78,7 @@ class VectorizeIndexImpl implements Vectorize {
         {
           method: "POST",
           body: JSON.stringify({
-            ...options,
+            ...(options ?? {}),
```

Comment:
this shouldn't be necessary, undefined and null can spread into an object

Reply:
Removed the check